docs/troubleshooting-guide/sandboxing.md

```diff
@@ -0,0 +1,56 @@
+# Sandboxing
+
+- Sandboxing means "giving some people access to only some data"
+- Can be set up [graphically][sandboxing-your-data]
+- Metabase runs a query that filters rows and/or selects a subset of columns
+- The user's query then runs on that
+
+So instead of:
+
+```
+select * from orders
+where price > 100.00;
+```
+
+we create a [common table expression][cte] as a replacement starting point for the query:
+
+```
+with temp01 as
+  (select *
+   from orders
+   where orders.customer_id = {{user_id}})
+select * from temp01
+where price > 100.00;
+```
+
+where `user_id` is bound to a property from the user record.
+
+## Troubleshooting Process
+
+### My question can't be sandboxed
+
+1. Public questions can't be sandboxed: if someone doesn't have to log in to view the question, Metabase doesn't have user properties or group properties available for filtering the data.
```

Comment:
and they will be able to see ALL the data

Reply:
incorporated - thanks